web-ui/src/main/resources/catalog/locales/fr-admin.json

```diff
@@ -1138,6 +1138,8 @@
     "ui-mod-editor": "Éditeur",
     "ui-mod-authentication": "Authentification",
     "ui-mod-workflowHelper": "Assistance au cycle de vie",
+    "ui-mod-limitedBatchEditing": "Edition par lots limitée",
```

Comment:
I believe French and other languages are supplied by Transifex so I don't think you should be supplying these changes in this PR.